docs/release_notes/flare_250.rst

```diff
@@ -143,3 +143,116 @@ Developer Tutorial Page
 To let users quickly learn Federated Learning with FLARE, we developed a `tutorial web page <https://nvidia.github.io/NVFlare>`_ with
 both code and video to interactively learn how to convert and run FL in a few minutes. We also
 created a tutorial catalog to help you easily search and find the examples you are interested in.
+
+**********************************
+Migration to 2.5.0: Notes and Tips
+**********************************
+
+FLARE 2.5.0 introduces some API and behavior changes. This migration guide will help you to migrate from the previous NVFlare version
+to the current version.
+
+
+XGBoost v1 - v2
+===============
+
+XGBoost support is enhanced in 2.5.0 to support secure training using Homomorphic Encryption (HE). The user interface is also simplified by
+setting the XGBoost parameters in the controller so all clients get the same parameters. 
+
+The main changes are:
+
+  - The xgboost params have been moved from the client configuration to server.
+  - New split_mode and secure_training parameters
+  - New :class:`SecureDataLoader<nvflare.app_opt.xgboost.histogram_based_v2.secure_data_loader.SecureDataLoader>`
```

Comment:
this class is renamed: https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_opt/xgboost/histogram_based_v2/csv_data_loader.py